aws_lambda_powertools/utilities/idempotency/persistence/base.py

```diff
@@ -135,6 +137,8 @@ def __init__(
             Function to use for calculating hashes, by default md5.
         raise_on_no_idempotency_key: bool, optional
             Raise exception if no idempotency key was found in the request, by default False
+        jmespath_options : Dict
+            Alternative JMESPath options to be included when filtering expr
```

Comment:
no-op: leaving a note to remember to move all of these outside persistence, so we keep persistence layer with one task only.